package.json

```diff
@@ -35,11 +35,10 @@
     "collect-metrics": "./scripts/collect-metrics.sh"
   },
   "devDependencies": {
-    "@babel/cli": "7.0.0-beta.44",
-    "@babel/core": "^7.0.0-beta.44",
-    "@babel/polyfill": "7.0.0-beta.44",
-    "@babel/preset-env": "7.0.0-beta.44",
-    "@babel/preset-es2015": "7.0.0-beta.44",
+    "@babel/cli": "7.0.0-beta.56",
```

Comment:
I believe `rc.1` is out now.

Reply:
Merged with https://github.com/uber/deck.gl/pull/2185